otterdog/adoptium.jsonnet

```
@@ -264,6 +264,47 @@ orgs.newOrg('adoptium') {
         },
       ],
     },
+    orgs.newRepo('marketplace-api.adoptium.net') {
+      allow_merge_commit: true,
+      allow_squash_merge: false,
+      allow_update_branch: false,
+      delete_branch_on_merge: false,
+      dependabot_alerts_enabled: false,
```

Consider allowing security updates to be reported. (I don't know if that is default already?)

```suggestion
      dependabot_alerts_enabled: false,
      dependabot_security_updates_enabled: true,
```